content/undergraduate/sci.md

```diff
@@ -0,0 +1,92 @@
+{#cc_top}
+
+# List of Science Elective Courses
+
+{#science-pool}
+
+## Science Pool
+
+Science electives must be selected from the lists below.
+
+{#bio}
+
+### BIO - Molecular Biology and Genetics
+
+:::{csv-table}
+:width: 100%
+:widths: 10 40 50
+:header-rows: 1
+
+"Code", "Course Title", "Notes"
+"BIOXXX", "All 3- or 4-credits non-laboratory BIOXXX courses", ""
```

Comment:
I don't like half dashes and may create ambiguity with numbers.

"3 or 4 credits ..."

Reply:
Done.